po/fr.po

```diff
@@ -1,5 +1,723 @@
+msgid "Unlock Layer"
+msgstr "Dévérouiller le calque"
+
+#: src/Layouts/Partials/Artboard.vala:529 src/Layouts/Partials/Layer.vala:729
+msgid "Show Layer"
+msgstr "Afficher le calque"
+
+#: src/Layouts/Partials/BorderItem.vala:121
+msgid "Choose border color"
+msgstr "Choisir la couleur de la bordure"
+
+#: src/Layouts/Partials/BorderItem.vala:176
+#: src/Layouts/Partials/FillItem.vala:178
+msgid "Pick color"
+msgstr "Choisir une couleur"
+
+#: src/Layouts/Partials/BorderItem.vala:191
+msgid "Remove border"
+msgstr "Supprimer la bordure"
+
+#: src/Layouts/Partials/BorderItem.vala:275
+msgid "Show border"
+msgstr "Afficher la bordure"
+
+#: src/Layouts/Partials/BorderItem.vala:279
+msgid "Hide border"
+msgstr "Masquer la bordure"
+
+#: src/Layouts/Partials/BorderRadiusPanel.vala:86
+msgid "Style"
+msgstr "Style"
+
+#: src/Layouts/Partials/BorderRadiusPanel.vala:103
+msgid "Border Radius"
+msgstr "Arrondi"
+
+#: src/Layouts/Partials/BorderRadiusPanel.vala:137
+msgid "Border radius options"
+msgstr "Options de l'arrondi"
+
+#: src/Layouts/Partials/BorderRadiusPanel.vala:199
+msgid "Autoscale Corners"
+msgstr "Mise à l'échelle automatique des coins"
+
+#: src/Layouts/Partials/BorderRadiusPanel.vala:208
+msgid "Uniform Corners"
+msgstr "Uniformiser les bordures"
+
+#: src/Layouts/Partials/BordersPanel.vala:53
+msgid "Borders"
+msgstr "Bordures"
+
+#: src/Layouts/Partials/BordersPanel.vala:64
+msgid "Add border"
+msgstr "Ajouter une bordure"
+
+#: src/Layouts/Partials/FillItem.vala:114
+msgid "Choose fill color"
+msgstr "Choisir la couleur de remplissage"
+
+#: src/Layouts/Partials/FillItem.vala:193
+msgid "Remove fill color"
+msgstr "Supprimer la couleur de remplissage"
+
+#: src/Layouts/Partials/FillItem.vala:277
+msgid "Show fill color"
+msgstr "Afficher la couleur de remplissage"
+
+#: src/Layouts/Partials/FillItem.vala:281
+msgid "Hide fill color"
+msgstr "Masquer la couleur de remplissage"
+
+#: src/Layouts/Partials/FillsPanel.vala:54
+msgid "Fills"
+msgstr "Remplissages"
+
+#: src/Layouts/Partials/FillsPanel.vala:65
+msgid "Add fill color"
+msgstr "Ajouter une couleur de remplissage"
+
+#: src/Layouts/Partials/TransformPanel.vala:96
+msgid "X"
+msgstr "X"
+
+#: src/Layouts/Partials/TransformPanel.vala:96
+msgid "Horizontal position"
+msgstr "Position horizontale"
+
+#: src/Layouts/Partials/TransformPanel.vala:98
+msgid "Y"
+msgstr "Y"
+
+#: src/Layouts/Partials/TransformPanel.vala:98
+msgid "Vertical position"
+msgstr "Position verticale"
+
+#: src/Layouts/Partials/TransformPanel.vala:100
+msgid "W"
+msgstr "W"
+
+#: src/Layouts/Partials/TransformPanel.vala:100
+msgid "Width"
+msgstr "Largeur"
+
+#: src/Layouts/Partials/TransformPanel.vala:102
+msgid "H"
+msgstr "H"
+
+#: src/Layouts/Partials/TransformPanel.vala:102
+msgid "Height"
+msgstr "Hauteur"
+
+#: src/Layouts/Partials/TransformPanel.vala:107
+msgid "Lock Ratio"
+msgstr "Ratio de verrouillage"
+
+#: src/Layouts/Partials/TransformPanel.vala:114
+msgid "R"
+msgstr "R"
```

Comment:
I need context to valid this one (poke @albfan)

Reply:
@Alecaddd could you give me a screenshot for this to check if the translation is the good one?